src/Chapter1.hs

```diff
@@ -490,8 +490,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
-
+lastDigit :: Integral a => a -> a
+lastDigit = flip mod 10 . abs
```

Comment:
Nice point-free implementation! Alternatively, you implement this function using infix form of calling functions in Haskell:

```suggestion
lastDigit n = abs n `mod` 10
```